### DISCUSSION TITLE
Subclassing PlainRegistry (but no access through `facets`)

### DISCUSSION THREAD

**Discussion by butlercj** [hgrecco/pint#1763] [General]
When [describing how to make a custom registry](https://pint.readthedocs.io/en/stable/advanced/custom-registry-class.html), it is said that we can subclass pint's PlainRegistry (and other kinds of registry) like this:
```
import pint
class MyRegistry(pint.facets.NonMultiplicativeRegistry, pint.facets.PlainRegistry):
    pass
```
But this returns "AttributeError: module 'pint' has no attribute 'facets'."

I guess this is because pint's `__init__` doesn't import the facets module. Instead, objects defined in facets are accessed through `pint.registry`, where `from .facets import ...` is used.

These objects (e.g. UnitRegistry, MeasurementRegistry, etc.) are subclasses of PlainRegistry, but PlainRegistry itself is defined in `pint.facets.plain.registry`, which we cannot access.

At this point I am stuck and I can't see how to subclass it. How can it be done?

Thank you!

**Comment by hgrecco** [comment]
We are revampin the subclassing experience to make it easier to use. The new version (coming out this week) should have a better roadmap!
